src/QsCompiler/LlvmBindings/LLVMSharpExtensions/LLVMValueRefExtensions.cs

```diff
@@ -5,37 +5,39 @@
 
 namespace LLVMSharp.Interop
 {
-    /// <summary>Extensions for the LLVMSharp.Interop.LLVMValueRef class</summary>
+    /// <summary>Extensions for the LLVMSharp.Interop.LLVMValueRef class.</summary>
     public static unsafe class LLVMValueRefExtensions
     {
         /// <summary>Convenience wrapper for LLVM.GetNextGlobalAlias.</summary>
-        public static LLVMValueRef NextGlobalAlias( this LLVMValueRef self ) => ( self.Handle != default ) ? LLVM.GetNextGlobalAlias( self ) : default;
+        public static LLVMValueRef NextGlobalAlias(this LLVMValueRef self) => (self.Handle != default) ? LLVM.GetNextGlobalAlias(self) : default;
 
-        /// <summary>Convenience wrapper for LLVM.GetNextGlobalIFunc</summary>
-        public static LLVMValueRef NextGlobalIFunc( this LLVMValueRef self ) => ( self.Handle != default ) ? LLVM.GetNextGlobalIFunc( self ) : default;
+        /// <summary>Convenience wrapper for LLVM.GetNextGlobalIFunc.</summary>
+        public static LLVMValueRef NextGlobalIFunc(this LLVMValueRef self) => (self.Handle != default) ? LLVM.GetNextGlobalIFunc(self) : default;
 
-        /// <summary>Convenience wrapper for LLVM.GetMDString</summary>
-        public static string GetMDString( this LLVMValueRef self )
+        /// <summary>Convenience wrapper for LLVM.GetMDString.</summary>
+        public static string GetMDString(this LLVMValueRef self)
         {
             var mdString = self.IsAMDString;
-            if ( mdString.Handle == default )
+            if (mdString.Handle == default)
             {
                 return string.Empty;
             }
+
             uint len;
-            var pStr = LLVM.GetMDString( mdString, &len );
-            if ( pStr == default )
+            var pStr = LLVM.GetMDString(mdString, &len);
+            if (pStr == default)
             {
                 return string.Empty;
             }
-            return ( new ReadOnlySpan<byte>( pStr, (int)len ) ).AsString( );
+
+            return new ReadOnlySpan<byte>(pStr, (int)len).AsString();
         }
 
-        /// <summary>Convenience wrapper for LLVM.GlobalCopyAllMetadata</summary>
-        public static (LLVMValueMetadataEntryRef metadataRef, uint count) GlobalCopyAllMetadata( this LLVMValueRef self )
+        /// <summary>Convenience wrapper for LLVM.GlobalCopyAllMetadata.</summary>
+        public static (LLVMValueMetadataEntryRef MetadataRef, uint Count) GlobalCopyAllMetadata(this LLVMValueRef self)
         {
             UIntPtr count;
-            LLVMValueMetadataEntryRef metadataRef = LLVM.GlobalCopyAllMetadata( self, &count);
+            LLVMValueMetadataEntryRef metadataRef = LLVM.GlobalCopyAllMetadata(self, &count);
             return (metadataRef, (uint)count);
         }
```

Comment:
Minor: Down below here is an old method that was accidentally left in but commented out, which can be safely removed.

Reply:
Removed, thank you.